OutSystems/outsystems-datagrid-reactive
# RGRIDT 723 - Improve the public API with more actions II

### What was happening
* Public methods related with styling were not implemented on the API

### What was done
* Created the _**SetCellCssClass**_ method;
* Created the _**SetColumnCssClass**_ method;
* Created the _**RemoveAllCssClassesFromCell**_ method;
* Created the _**RemoveColumnCssClass**_ method;

### Test Steps
1. Add an onChange client action on one column;
2. Use the **AddCellCssClass** API action to change other cell appearance;
3. Add another onChange client action on another column;
4. Use the **RemoveAllCssClassesFromCell** API action to change other cell appearance;
5. Interact with the grid and check if the class is added on the first moment;
6. Then check if it's removed;
#
1. Add an onChange client action on one column;
2. Use the **AddColumnCssClass** API action to change other collumn appearance;
3. Add another onChange client action on another column;
4. Use the **RemoveCssClassesFromColumn** API action to change other collumn appearance with **_ApplyToHeader_** parameter (false);
5. Interact with the grid and check if the class is added on the first moment but the header cell of the column is not affected;
6. Then check if it's removed;
#
1. Add an onChange client action on one column;
2. Use the **AddColumnCssClass** API action to change other collumn appearance with **_ApplyToHeader_** parameter (true);
3. Add another onChange client action on another column;
4. Use the **RemoveCssClassesFromColumn** API action to change other collumn appearance;
5. Interact with the grid and check if the class is added on the first moment including the header cell;
6. Then check if it's removed;

### Checklist
